doc/api/http.md

```diff
@@ -832,8 +836,10 @@ Listener of this event is responsible for closing/destroying the underlying
 socket. For example, one may wish to more gracefully close the socket with a
 custom HTTP response instead of abruptly severing the connection.
 
-Default behavior is to close the socket with an HTTP '400 Bad Request' response
-if possible, otherwise the socket is immediately destroyed.
+Default behavior is to try close the socket with a HTTP '400 Bad Request',
```

Comment:
Is the `try` necessary in this sentence?

Reply:
I added `try` to be synonymous with the `if possible` of the original sentence, that being if the socket is not writable it can't do the default HTTP status code behaviour, and closes immediately.

From master:

https://github.com/nodejs/node/blob/5571c0ddebe72e894e41bbee56c7165a55788288/lib/_http_server.js#L515-L520

Another way to word it could be:

> Default behavior for a writable socket is to close it with a HTTP '400 Bad Request' or a HTTP '431 Request Header Fields Too Large' in the case of a
[`HPE_HEADER_OVERFLOW`][] error. If the socket is not writable it is
immediately destroyed.